spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/actions/ManifestFileBean.java

```diff
@@ -98,6 +100,10 @@ public void setSequenceNumber(Long sequenceNumber) {
     this.sequenceNumber = sequenceNumber;
   }
 
+  public void setFirstRowId(Long firstRowId) {
+    this.firstRowId = firstRowId;
+  }
+
   @Override
```

Comment:
I think we should get in #13061 first, then I can rebase and address this (the Avro test already tests distributed planning)